app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    result_cache = {}
+
+    def inner(*args) -> Callable:
```

Comment:
```suggestion
    def inner(*args, **kwargs) -> Any:
```

- Use `functools.wraps`
- What if the function accepts keyword arguments? 
- Wrong type annotation